src/main/java/com/SBS/Services/MatchService.java

```diff
@@ -0,0 +1,40 @@
+package com.SBS.Services;
+
+import com.SBS.Models.Match;
+import com.SBS.Models.MatchResult;
+import com.SBS.Repositories.MatchRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MatchService {
+    MatchRepository matchRepository;
+
+    MatchService(MatchRepository matchRepository) {
+        this.matchRepository = matchRepository;
+    }
+
+    public Match addMatch(Match match) {
+        return matchRepository.addMatch(match);
+    }
+
+    public void removeMatch(Long id) {
+        matchRepository.removeMatch(id);
+    }
+
+    public Match getMatchById(Long id) {
+        return matchRepository.getMatchById(id);
+    }
+
+    public List<Match> getMatches() {
+        return matchRepository.getMatches();
+    }
+
+    public Match setResult(Long id,MatchResult matchResult)
```

Comment:
to setResult nie zapisze się na bazie danych, tylko bedzie obecne w zwrotce. Aby to setResult mialo zastosowanie na db to mozesz uzyc @Transactional (tutaj doczytaj bo to nie jest lekki temat) albo zrob repo.save()

Reply:
zgoda